contracts/burner/README.md

```diff
@@ -0,0 +1,13 @@
+# Burner Contract
+
+This is a simple contract to demonstrate using migrations to
+shutdown (or "burn") contracts using the new migration feature.
```

Comment:
This text will get outdated soon. Better: "the migration feature added in CosmWasm 0.9"